rust/otap-dataflow/crates/otap/src/syslog_cef_receiver.rs

```diff
@@ -110,43 +123,63 @@ pub static SYSLOG_CEF_RECEIVER: ReceiverFactory<OtapPdata> = ReceiverFactory {
 #[async_trait(?Send)]
 impl local::Receiver<OtapPdata> for SyslogCefReceiver {
     async fn start(
-        self: Box<Self>,
+        mut self: Box<Self>,
         mut ctrl_chan: local::ControlChannel<OtapPdata>,
         effect_handler: local::EffectHandler<OtapPdata>,
     ) -> Result<(), Error> {
+        // Start periodic telemetry collection (1s), similar to other nodes
+        let handle = effect_handler
+            .start_periodic_telemetry(std::time::Duration::from_secs(1))
+            .await?;
+        let mut telemetry_timer = Some(handle);
```

Comment:
Why are we wrapping it with an option? I don't think it's needed. You can use the handle directly.

```rust
let timer_cancel_handle = effect_handler
    .start_periodic_telemetry(Duration::from_secs(1))
    .await?;

...

Ok(NodeControlMsg::Shutdown { .. }) => {
    _ = timer_cancel_handle.cancel().await;
    break;
}
```